lib/wrap.js

```diff
@@ -56,7 +56,13 @@ function metrics(main) {
         }
 
         try {
-            return await main(params);
+            // send activation_start metrics and execute main action in parallel
+            const activationStartedMetricsPromise = metrics.activationStarted();
+            const mainPromise = main(params);
+            await Promise.all([activationStartedMetricsPromise, mainPromise]);
```

Comment:
Another question - I wonder if we should ensure order here, aka first run the `activationStartedMetricsPromise`, and then the `main` promise (use the keyword "then" to chain them). Because, in theory, with `Promise.all`, the main promise can finish before the NR one does (not highly probable, but not impossible either). Wdyt?